app/controllers/chapters_controller.rb

```diff
@@ -177,7 +177,7 @@ def update
 
   def update_positions
     if params[:chapters]
-      @work.reorder(params[:chapters])
+      @work.reorder(params[:chapters].map { |c| c.to_i.to_s })
```

Comment:
I'm pretty sure this is the same `reorder` issue as above, and could be addressed by renaming the function to `reorder_chapters` instead.